libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/VerifiableCredentialParser.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.di.ipv.core.library.helpers;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jwt.JWTClaimsSet;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.gov.di.ipv.core.library.exceptions.CredentialParseException;
+import uk.gov.di.model.AddressCredential;
+import uk.gov.di.model.IdentityAssertionCredential;
+import uk.gov.di.model.IdentityCheckCredential;
+import uk.gov.di.model.RiskAssessmentCredential;
+import uk.gov.di.model.SecurityCheckCredential;
+import uk.gov.di.model.VerifiableCredential;
+import uk.gov.di.model.VerifiableCredentialType;
+
+import java.text.ParseException;
+import java.util.List;
+
+public class VerifiableCredentialParser {
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final ObjectMapper LAX_OBJECT_MAPPER =
+            new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+    private VerifiableCredentialParser() {}
+
+    public static VerifiableCredential parseCredential(JWTClaimsSet claimsSet)
+            throws CredentialParseException {
+        try {
+            var vcClaim = claimsSet.getJSONObjectClaim("vc");
```

Comment:
```suggestion
            var vcClaim = claimsSet.getJSONObjectClaim(VC_CLAIM);
```
We have a bunch of constants defined in `VerifiableCredentialConstants`